kata-deploy/scripts/kata-deploy.sh

```diff
@@ -100,7 +100,7 @@ EOT
 	fi
 
         # add kata-qemu-virtiofs config
-	if grep -q "\[$kata_qemu_virtiofs_conf\]" $crio_conf_file; then
+	if grep -qEe "^\[$kata_qemu_virtiofs_conf\]" $crio_conf_file; then
```

Comment:
Do we actually need the `-e` here and below? The man page suggests it's only useful if the pattern starts with a dash (which it doesn't?)

Reply:
it was discussed. We technically don't need either the `E` or `e` for gnu grep - but, adding them in makes it more portable across other distro/grep's was the base theory.